File: Test Code/ScoresCalculator_CSharp/AdjacentDepartment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ScoresCalculator
{
    public class AdjacentDepartment
    {
        public List<Department> Departments = new List<Department>();
        public double ClosenessRating { set; get; }

        public AdjacentDepartment(Department firstDepartment, Department secondDepartment, double score)
        {
            Departments.Add(firstDepartment);
            Departments.Add(secondDepartment);
            ClosenessRating = score;
        }

        public bool CheckDepartments(Department firstDepartment, Department secondDepartment)
        {
            if (Departments[0].Index == firstDepartment.Index && Departments[1].Index == secondDepartment.Index)
            {
                return true;
            }
            else if (Departments[0].Index == firstDepartment.Index && Departments[1].Index == secondDepartment.Index)
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }
}

File: Test Code/AdjacencyCalculator/AdjacentDepartment.java
package src;

public class AdjacentDepartment {
	Department[] Departments = new Department[2];
	public double ClosenessRating;

    public AdjacentDepartment(Department firstDepartment, Department secondDepartment, double score)
    {
        Departments[0] = firstDepartment;
        Departments[1] = secondDepartment;
        ClosenessRating = score;
    }

    public boolean CheckDepartments(Department firstDepartment, Department secondDepartment)
    {
        if (Departments[0].Index == firstDepartment.Index && Departments[1].Index == secondDepartment.Index)
        {
            return true;
        }
        else if (Departments[1].Index == firstDepartment.Index && Departments[0].Index == secondDepartment.Index)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}
